awesome-list.md

```diff
@@ -8,6 +8,7 @@
 
 ## Awesome List
 
+- [Catch 'Em All App](https://pokemon.ptdede.me/) by [Your name](/your-github-link): NextJS, Apollo GraphQL, Typescript, IDB.
```

Comment:
Update `/your-github-link` if you can

Reply:
Can I remove the github link? 😄 🙏
@mazipan

Reply:
Of course, if you didn't comfort to expose the code